docs/sdk-scheme-adapter-outbound-v2_0_0-openapi3-snippets.yaml

```diff
@@ -2370,7 +2370,8 @@ components:
     requestToPayTransferRequest:
```

Comment:
What about:

1. `retriesLeft` on the `PUT /requestToPayTransfer` on the request?
2. `authResponse` on the `PUT /requestToPayTransfer` sync response?

Reply:
As discussed the changes would be updated in the OTP follow-up story.

So the PR LGTM.

Reply:
Thanks